### PR TITLE
A11y/use fieldsets for prefs

### PR DESCRIPTION
**Pull request by gcamacho079**
### Description
Use fieldsets for accessibility and dev preferences (Resolves DEV-333).

Also fixes an issue where `fieldset` elements are nested inside `fieldset` elements for both radio groups and checkbox groups.

### CODE REVIEW

File: src/templates/_includes/forms/checkboxGroup.twig [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 {% set fieldsetAttributes = {
     class: ['checkbox-group'],
+    role: 'presentation',
```
Comment (gcamacho079) [inline src/templates/_includes/forms/checkboxGroup.twig:7]: @brandonkelly I added this for both the `checkboxGroup` and the `radioGroup` as I wasn't sure if changing the `fieldset` element to a `div` would be problematic; if not let me know and I can modify the tag itself.

Comment (brandonkelly) [reply src/templates/_includes/forms/checkboxGroup.twig:7]: Probably safe to just change the tag.

Comment (gcamacho079) [reply src/templates/_includes/forms/checkboxGroup.twig:7]: Will do!

**Comment by brandonkelly** [comment]
👍🏻 Replaced “Location” with “General”, and also split the Debug Toolbar-related development settings into their own checkbox group.

The new user preferences form, with 2nd level headings “General”, “Accessibility”, and “Development”

**Pull request merged**